### PR TITLE
IDEX-3637 Display build logs in OpenShift output panel

### PR DESCRIPTION
**Pull request by vitaliy-guliy**

### CODE REVIEW

File: che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/BuildsPresenter.java [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+    /**
+     * Checks workspace projects for being deployed on OpenShift and starts observation for builds for that projects.
+     */
+    private void checkWorkspaceProjects() {
+        projectServiceClient.getProjects(false).then(new Operation<List<ProjectConfigDto>>() {
+            @Override
+            public void apply(List<ProjectConfigDto> projects) throws OperationException {
+                for (ProjectConfigDto project : projects) {
+                    if (project.getMixins().contains("openshift") &&
+                            project.getAttributes().containsKey(OpenshiftProjectTypeConstants.OPENSHIFT_NAMESPACE_VARIABLE_NAME)) {
```
Comment (sleshchenko) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/BuildsPresenter.java]: It is required variable. So If project has mixin openshift it must have this variable

Comment (vitaliy-guliy) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/BuildsPresenter.java]: Ok

File: che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/StartBuildAction.java [hunk 2]
```diff
@@ -48,22 +48,23 @@
     private final OpenshiftServiceClient        openshiftService;
     private final NotificationManager           notificationManager;
     private final OpenshiftLocalizationConstant locale;
-    private final BuildStatusWatcher            buildStatusWatcher;
+
```
Comment (sleshchenko) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/StartBuildAction.java]: What is special in BuildsPresenter? Why do you separate it from other fields with empty line?

File: che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/project/wizard/CreateProjectPresenter.java [hunk 3]
```diff
@@ -94,6 +100,8 @@ public void onCreateClicked() {
         wizard.complete(new Wizard.CompleteCallback() {
             @Override
             public void onCompleted() {
+                final String namespace = wizard.getDataObject().getProjectConfigDto().getAttributes().get(OPENSHIFT_NAMESPACE_VARIABLE_NAME).get(0);
```
Comment (sleshchenko) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/project/wizard/CreateProjectPresenter.java:103]: potential NPE or ArrayIndexOutOfBoundsException

File: che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/importapp/ImportApplicationPresenter.java [hunk 4]
```diff
@@ -315,6 +322,18 @@ protected void onSuccess(ProjectConfigDto result) {
                     view.setBlocked(false);
                     view.closeView();
 
+                    try {
+                        final String namespace = result.getAttributes().get(OPENSHIFT_NAMESPACE_VARIABLE_NAME).get(0);
```
Comment (sleshchenko) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/importapp/ImportApplicationPresenter.java]: potential NPE or ArrayIndexOutOfBoundsException

File: che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/OpenshiftGinModule.java [hunk 5]
```diff
@@ -23,6 +25,7 @@
     @Override
     protected void configure() {
         bind(OpenshiftServiceClient.class).to(OpenshiftServiceClientImpl.class);
+        bind(BuildsView.class).to(BuildsViewImpl.class);
```
Comment (sleshchenko) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/OpenshiftGinModule.java]: What do you think about usage of @ImplementedBy annotation?

**Comment by sleshchenko** [comment]
OK for me

File: che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/StartBuildAction.java (hunk 2 above)
Comment (vitaliy-guliy) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/StartBuildAction.java]: Fixed

File: che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/importapp/ImportApplicationPresenter.java (hunk 4 above)
Comment (vitaliy-guliy) [inline che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/importapp/ImportApplicationPresenter.java]: Fixed

**Pull request merged**